prow/jobs/kyma-project/compass-manager/compass-manager-tests.yaml

```diff
@@ -0,0 +1,34 @@
+presubmits: # runs on PRs
+  kyma-project/compass-manager:
+    - name: pre-main-compass-manager-tests
+      annotations:
+        pipeline.test: "fast-integration"
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-compass-manager-tests"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      optional: false
+      run_if_changed: '^(go.mod|go.sum)$|^*/(.*.go|Makefile|Dockerfile|.*.sh)'
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/compass-manager
```

Comment:
I don't think you need this line. The workdir automatically is set to your source.